src/devices/sound/spu.h

```diff
@@ -145,6 +143,9 @@ class spu_device : public device_t, public device_sound_interface
 	static reverb_preset reverb_presets[];
 	static reverb_params *spu_reverb_cfg;
 
+	const float ms_to_rate(float ms) { return 1.0f / (ms * (spu_base_frequency_hz / 1000.0f)); }
+	const float s_to_rate(float s) { return ms_to_rate(s * 1000.0f); }
+
```

Comment:
Not like this – `const` return-by-value has no effect.  You need to do `float ms_to_rate(float ms) const ...` etc.